### ISSUE TITLE
ADAL should resolve TeamId() at runtime

### ISSUE THREAD

**Issue by jennyf19** [AzureAD/azure-activedirectory-library-for-dotnet#1564]
ADAL should resolve the iOS TeamId() at runtime for keychain access. the TeamId prefix will change between dogfood and development, so enabling ADAL to resolve the TeamId() at runtime is crucial and will match behavior of MSAL 3.x 

Propose a new property on authenticationContext:
`iOSKeychainSecurityGroup`

to use like this:
```CSharp
AuthenticationContext authContext = new AuthenticationContext(_authority);
#if iOS
     authContext.iOSKeychainSecurityGroup = "com.microsoft.msalrocks";
#endif
```
cc: @jmprieur

**Comment by jennyf19** [comment]
addressed in this [pr](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1562) and have updated [wiki](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/wiki/iOS-Keychain-Access) + aka.ms link

**Comment by jennyf19** [comment]
released in [ADAL 5.0.1-preview](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/releases/tag/ADAL-v5.0.1-preview)
